composer.json

```diff
@@ -30,6 +32,18 @@
         "phpunit/phpunit": "^9.6",
         "squizlabs/php_codesniffer": "^3"
     },
+    "extra": {
+        "project-files-installed": [
+            ".htaccess",
+            "app/.htaccess",
+            "app/_config/mysite.yml"
+        ],
+        "public-files-installed": [
+            ".htaccess",
+            "index.php",
+            "web.config"
+        ]
+    },
```

Comment:
Looks like you've removed some stuff from here? All files from recipes need to be listed here, or they'll just be added back.
Please see https://github.com/silverstripe/rhino/blob/main/composer.json for an accurate example.

Any files we don't want, we just don't commit the file - but the file still has to be referenced here to not be copied into the project.

Reply:
Page + PageController won't be copied in, because we're not using recipe-cms, we're using recipe-core. 

app/_config/mimevalidator.yml should have been in there though, have added back in